src/AppBundle/Mailjet/Message/CommitteeContactMembersMessage.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AppBundle\Mailjet\Message;
+
+use AppBundle\Entity\Adherent;
+use Ramsey\Uuid\Uuid;
+
+final class CommitteeContactMembersMessage extends MailjetMessage
+{
+    /**
+     * @param Adherent[] $recipients
+     * @param Adherent   $animator
+     * @param string     $message
+     *
+     * @return CommitteeContactMembersMessage
+     */
+    public static function create(array $recipients, Adherent $animator, string $message): self
+    {
+        $message = new self(
+            Uuid::uuid4(),
+            '54909',
+            $animator->getEmailAddress(),
+            $animator->getFullName(),
+            "L'animateur d'un comité que vous suivez vous a envoyé un message",
+            ['animator_firstname' => $animator->getFirstName(), 'animator_message' => $message],
+            ['target_firstname' => $animator->getFirstName()]
+        );
```

Comment:
The last argument must be given. It's a UUID that must be the same for all recipients. This value allows us to know which email are sent in the same batch of recipients.